kitsune/users/jinja2/users/includes/login-form.html

```diff
@@ -1,6 +1,23 @@
 {% from "users/includes/macros.html" import login_form as render_login with context %}
 
-<article id="login" class="main">
-  <h1>{{ _('Sign In') }}</h1>
+<article id="login" class="main login-form">
+  <h1 class="mb-0">{{ _('Sign In') }}</h1>
+  <p class="mt-0 mb-0">or <a href="#TODO">continue with Firefox Account</a></p>
   {{ render_login(login_form, next_url=next_url) }}
+
+  <br>
+  <br>
+  <br>
+  <br>
+  <h1 class="mb-0">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
+  <p class="mt-0">We are upgrading our log-in system to replace Mozilla Support accounts with Firefox Accounts. Firefox Accounts offers one log-in for convenient and secure access to Mozilla services. <a href="#TODO">See our FAQ</a> for more information.</p>
+
+  <div>
+  <a href="#TODO" class="button">Continue with Firefox Accounts</a>
```

Comment:
There is an implementation in a different branch:
https://github.com/mozilla/kitsune/blob/firefox-accounts-integration/kitsune/users/urls.py#L101

Maybe you can use this URL for consistency.

Reply:
That URL is present in the target branch, but not in this feature branch. My plan was to hook this page up to the proper URLs, and create new ones where necessary, once this screen is merged.

Alternatively, if you'd rather, I can rebase this branch on top of the target and make the change.

Reply:
Let's do that